src/opnsense/mvc/app/controllers/OPNsense/Kea/forms/dialogSubnet4.xml

```diff
@@ -145,4 +145,13 @@
             <visible>false</visible>
         </grid_view>
     </field>
+    <field>
+        <id>subnet4.option_data.v6_only_preferred</id>
+        <label>IPv6-only Preferred</label>
+        <type>text</type>
+        <help>The number of seconds for which the client should disable DHCPv4</help>
+        <grid_view>
+            <visible>false</visible>
+        </grid_view>
```

Comment:
perhaps add advanced here, using the following tag:

```
<advanced>true</advanced>
```

I doesn't feel like a very common option at a first glance, other than that, I'm fine with adding it